i18n/zh-tw.toml

```diff
@@ -30,7 +30,7 @@ other = "隱私政策"
 
 # Post (blog, articles etc.)
 [post_byline_by]
-other = "藉由"
+other = "作者："
```

Comment:
For consistency, I'd rather omit the punctuation here:

```suggestion
other = "作者"
```

Reply:
Actuall if you removed the `:` symbol, it reads odd in Chinese. I can live with that. I'm just try to make it perfect in Chinese expression.